homeassistant/core.py

```diff
@@ -190,6 +190,8 @@ def _async_add_job(self, target: Callable[..., None], *args: Any) -> None:
             self.loop.create_task(target)
         elif is_callback(target):
             self.loop.call_soon(target, *args)
+        elif isinstance(target, asyncio.Future):
```

Comment:
We can define that not allow to make a job from a base async function. I don't need that for this PR but I see the problem in working process